configs/components/facter.rb

```diff
@@ -171,6 +171,10 @@
     end
   end
 
+  # In PE, aio_agent_version is distinguished from aio_agent_build by not including the git sha.
+  # Strip it off; this should have no impact on final releases, as git sha would not be included.
+  aio_agent_version = settings[:package_version].match(/^\d+\.\d+\.\d+(\.\d+){0,2}/).to_s
```

Comment:
Oh, actually, the to_s of a MatchData, does that return just the matched portion?